### PR TITLE
Clean up some warnings and other console logs in frontend tests

### PR DESCRIPTION
**Pull request by jasalisbury**
## Description of change

While looking for flaky frontend tests to cleanup I got carried away trying to remove some noise from the test logs. Some prop types got cleaned up as well. There is still a lot of console logs to cleanup, decided to checkpoint here to keep the PRs smallish.

## How to test

Run tests locally, verify tests pass on CI

### CODE REVIEW

File: frontend/src/widgets/Overview.js [hunk 1]
```diff
@@ -83,14 +83,14 @@ Overview.propTypes = {
 };
 
 Overview.defaultProps = {
-  data: PropTypes.shape({
```
Comment (thewatermethod) [inline frontend/src/widgets/Overview.js:86]: Curious about this change - didn't realize PropTypes.shape was incorrect

File: frontend/src/pages/Landing/index.js [hunk 2]
```diff
@@ -228,7 +228,13 @@ function Landing({ user }) {
   const [alertReportsCount, setAlertReportsCount] = useState(0);
   const [filters, setFilters] = useState([]);
   const [alertFilters, setAlertFilters] = useState([]);
-  const [appliedRegion, updateAppliedRegion] = useState(user.homeRegionId === 14 ? 14 : regions[0]);
+
+  const defaultRegion = regions[0] || user.homeRegionId || 0;
```
Comment (thewatermethod) [inline frontend/src/pages/Landing/index.js:232]: Do we need the third option here (setting it to 0)?

**Review by thewatermethod — COMMENTED** [review]
This is a big improvement. Learned a lot about Jest just looking through these changes, and it'll be good to get the console output cleaned up. The fact that I'm still seeing a ton speaks to how much cleanup still remains (so again, this is great)!

I left questions, although neither would stop me from approving and are more things I'm interested in understanding. 

RE the CI failure - Both Adam and I have written some additional tests in branches that aren't merged yet, its possible you may be able to borrow some of those tests to get that branch coverage number up.

File: frontend/src/pages/Landing/index.js (hunk 2 above)
Comment (jasalisbury) [reply frontend/src/pages/Landing/index.js:232]: User's do not necessarily have a `homeRegionId`. the zero catches the case where the user isn't assigned any regions and does not have a `homeRegionId`

File: frontend/src/widgets/Overview.js (hunk 1 above)
Comment (jasalisbury) [reply frontend/src/widgets/Overview.js:86]: I did too when I first wrote it! :upside_down_face:

**Review by thewatermethod — APPROVED** [review]

**Pull request merged**